native/cocos/renderer/pipeline/custom/NativeRenderGraph.cpp

```diff
@@ -772,6 +773,12 @@ void NativeRenderQueueBuilder::addGpuDrivenResource(const scene::Camera *camera,
             ccstd::pmr::string drawInstanceBuffer("CCDrawInstanceBuffer");
             drawInstanceBuffer.append(std::to_string(cullingID));
 
+            const auto objectNameID = layoutGraph->attributeIndex.at("CCObjectBuffer");
+            data.bufferNames[objectNameID.value] = objectBuffer;
```

Comment:
Might remove this buffer from scene node

Reply:
1